accesspoc/templates/includes/dc.html

```diff
@@ -1,18 +1,5 @@
-{% load i18n %}
-
 {% if dc %}
-  <p><strong>{% trans "Identifier" %}:</strong> {{ dc.identifier }}</p>
-  <p><strong>{% trans "Title" %}:</strong> {{ dc.title }}</p>
-  <p><strong>{% trans "Date" %}:</strong> {{ dc.date }}</p>
-  <p><strong>{% trans "Creator" %}:</strong> {{ dc.creator }}</p>
-  <p><strong>{% trans "Format" %}:</strong> {{ dc.format|safe }}</p>
-  <p><strong>{% trans "Description" %}:</strong> {{ dc.description|safe }}</p>
-  <p><strong>{% trans "Subject" %}:</strong> {{ dc.subject }}</p>
-  <p><strong>{% trans "Publisher" %}:</strong> {{ dc.publisher }}</p>
-  <p><strong>{% trans "Contributor" %}:</strong> {{ dc.contributor }}</p>
-  <p><strong>{% trans "Type" %}:</strong> {{ dc.type }}</p>
-  <p><strong>{% trans "Source" %}:</strong> {{ dc.source }}</p>
-  <p><strong>{% trans "Language" %}:</strong> {{ dc.language }}</p>
-  <p><strong>{% trans "Coverage" %}:</strong> {{ dc.coverage }}</p>
-  <p><strong>{% trans "Rights" %}:</strong> {{ dc.rights }}</p>
+  {% for label, value in dc.get_display_data.items %}
+    <p><strong>{{ label|capfirst }}:</strong> {{ value }}</p>
+  {% endfor %}
```

Comment:
I wonder if class-based views (https://docs.djangoproject.com/en/2.1/topics/class-based-views/) would fit this use case. I think it's ok the way you're doing it, we don't have to do all the Django way, just out of curiosity, what do you think? I've never used them personally.

Reply:
I'm not sure that's something that could be implemented for a partial. This template is only used from the Collection and Folder view pages and those views make some ES work, so it may not worth the effort either to do it for those models at this point.